src/pool/pool_disjoint.cpp

```diff
@@ -57,13 +58,44 @@ typedef struct umf_disjoint_pool_shared_limits_t {
     std::atomic<size_t> TotalSize;
 } umf_disjoint_pool_shared_limits_t;
 
+// Configuration of Disjoint Pool
+typedef struct umf_disjoint_pool_params_t {
+    // Minimum allocation size that will be requested from the system.
+    // By default this is the minimum allocation size of each memory type.
+    size_t SlabMinSize;
+
+    // Allocations up to this limit will be subject to chunking/pooling
+    size_t MaxPoolableSize;
+
+    // When pooling, each bucket will hold a max of 'Capacity' unfreed slabs
```

Comment:
I guess, these comments could be moved to the API docs now...

Reply:
The header file already has these descriptions.
Should I remove it here?

Reply:
this comment got my attention because it was a little more different/verbose than the one from API docs, but I can see you've updated it - thx!

I'll leave it up to you if remove or keep them